packages/app-project/src/components/ProjectHeader/components/DropdownNav/DropdownNav.stories.js

```diff
@@ -0,0 +1,126 @@
+import zooTheme from '@zooniverse/grommet-theme'
+import { Grommet } from 'grommet'
+import { Provider } from 'mobx-react'
+import { applySnapshot } from 'mobx-state-tree'
+
+import DropdownNav from './DropdownNav.js'
+import initStore from '@stores'
+
+function ThemedStory(Story) {
```

Comment:
A suggestion for pairing stories with unit tests:
`composeStories` from `@storybook/testing-react` can be used to compose a fully formed story component, and I think ThemedStory wouldn't need to be defined in both files.

Reply:
That’s a great suggestion. I was wondering how to import the decorated stories. I wouldn’t need to manually declare props either.